assays/ISSAAC-seq/spec.yaml

```diff
@@ -0,0 +1,383 @@
+!Assay
+name: ISSAAC-seq
+doi: https://doi.org/10.1038/s41592-022-01601-4
+publication_date: 15 September 2020
+description: The ISSAAC-seq is developed based on the property that Tn5 can use DNA-DNA
+  and DNA-RNA as substrates.
+modalities:
+- RNA
+- ATAC
+lib_struct: https://teichlab.github.io/scg_lib_structs/methods_html/ISSAAC-seq.html
+assay_spec:
+# RNA modality
+- !Region
+  region_id: RNA
+  region_type: RNA
+  name: ISSAAC-seq RNA
+  sequence_type: joined
+  sequence: AATGATACGGCGACCACCGAGATCTACACNNNNNNNNNNNNNNNNTCGTCGGCAGCGTCAGATGTGTATAAGAGACAGXNNNNNNNNNNAGATCGGAAGAGCACACGTCTGAACTCCAGTCACNNNNNNNNATCTCGTATGCCGTCTTCTGCTTG
+  min_len: 155
+  max_len: 455
+  onlist: null
+  regions:
+  - !Region
+    parent_id: RNA
+    region_id: RNA-illumina_p5
+    region_type: illumina_p5
+    name: Illumina P5
+    sequence_type: fixed
+    sequence: AATGATACGGCGACCACCGAGATCTACAC
+    min_len: 29
+    max_len: 29
+    onlist: null
+    regions: null
+  - !Region
+    parent_id: RNA
+    region_id: RNA-I2.fastq.gz
+    region_type: fastq
+    name: Index 2
+    sequence_type: joined
+    sequence: NNNNNNNNNNNNNNNN
+    min_len: 16
+    max_len: 16
+    onlist: null
+    regions:
+      - !Region
```

Comment:
The indentation here needs to be fixed. The whole block should be left indented on stop (so that the "-" is aligned with the "regions" tag above) and the contents of the - !Region should be right indented on block so that the contents align with the !Region tag. 

Try running `seqspec print spec.yaml` from the command line and if this was fixed then you should get no errors.

Reply:
Thanks. That's very clear. Will change.